### ISSUE TITLE
ChromeXt front-end for multiple browsers

### ISSUE THREAD

**Issue by alanjsebas** [JingMatrix/ChromeXt#161]
I can't get the front end to show up when i visit the pages on the readme, all I get is a message that it's working.

No option to manage the userscripts....

**Comment by JingMatrix** [comment]
You need to install some scripts so that you Can manage *them*.
Did you have any scripts installed?

**Comment by Guybrush3pw00d** [comment]
Hi. I remember about using ChromeXT months ago to get a user script

https://greasyfork.org/it/scripts/2128-rai-play-video-download

working on Soul Browser. Everything went smoothly apart from the fact that the script installed fine and it worked too but only partially, maybe because of the missing support for some APIs (or maybe their slightly different implementation or even the disabled GM.ChromeXt API according to the readme; I haven't the needed skills to read the script code and pinpoint the reason so I gave up).
Today I wanted to try again and see if something has changed in the meantime and things got even worse instead. I'm on A13, rooted with HuskyDG's Kitsune (Zygisk enabled), LSPosed Zygisk 1.9.2, ChromeXT 3.8.1 installed as adviced, enabled in LSPosed for Kiwi and Soul as I wanted to check if the issus with that user script above happened to be linked with WebView browsers rather than with a matter of APIs support.
Either browser I try, when I try to install the script the Greasyfork.org page behaves like I haven't a script manager installed. Everything goes just fine if I install Tampermonkey on Kiwi (while I can't on Soul).
It seems to me something similar to the 'issue' (I tend to believe it isn't exactly that) outlined at the beginning and this is the reason why I've chosen not to open another ticket. 
Thanks in advance for your advices and even more for the developing efforts with ChromeXT and have a nice day! :)

**Comment by alanjsebas** [comment]
Actually it seems I was confused, I had installed the Userscripts separately for a different applcation that relies on Webview (Slimsocial for Twitter), i tried twitter userscripts and assumed the manager would work system wide for any applications the scripts were installed into.

The manager works as intended when I install it into the browser itself. Is there a way to remove scripts for the webview applications where u can't visit a custom url?

**Comment by JingMatrix** [comment]
> The manager works as intended when I install it into the browser itself. Is there a way to remove scripts for the webview applications where u can't visit a custom url?

That could be complicated. 
If you can open Eruda console, then you can open the manager page, and click `Info` -> `UserScripts` -> the eye icon.

**Comment by alanjsebas** [comment]
> > The manager works as intended when I install it into the browser itself. Is there a way to remove scripts for the webview applications where u can't visit a custom url?
> 
> That could be complicated. If you can open Eruda console, then you can open the manager page, and click `Info` -> `UserScripts` -> the eye icon.

Right next to the + sign an in blue?

It asks me if I want to load the manager page in my browser while clicking on that

**Comment by JingMatrix** [comment]
Yeap, you can then open the front end, I think.

**Comment by JingMatrix** [comment]
@Guybrush3pw00d I modfied a bit ChromeXt so that your mentioned script will work as expected.
Check it out in GitHub Actions.

**Comment by Guybrush3pw00d** [comment]
@JingMatrix 

Thanks! :) I'll try it ASAP and confirm it as working (I'm already sure about that). Have a nice day.

**Comment by Guybrush3pw00d** [comment]
I can confirm that the latest version downloadable on Github Actions installed just fine this time on Soul (WebView Browser) and it works as expected.
I'm experiencing issues on Kiwi instead, probably because I don't have Eruda but I need to better document myself about that.
If I try to download the script from Greasyfork, it doesn't offer me the option to install it but just download. I've then installed the downloaded file by the Kiwi add-on manager. It installs fine and is listed among the installed add-ons but it doesn't work. This isn't the right way to install for sure.
If I go [here,](https://jingmatrix.github.io/ChromeXt/ it says ChromeXT is working as I'd expect but it doesn't list the user script like it should (and it does just fine on Soul).
To summarise, Soul works a treat while I still need to work on Kiwi, almost surely because of mistakes on my side.
Have a nice day! :)

**Comment by JingMatrix** [comment]
@Guybrush3pw00d Please open a new issue next time.

I tested Kiwi just now. The situation is not as perfect as before, but still better than Edge #156 .
Currently, you can install scripts: 1. use Eruda to pick up the downloaded script file 2. open downloaded scripts from the download menu (to ensure the script is installed into kiwi, you may need to disable all other browsers, as explained in https://github.com/JingMatrix/ChromeXt?tab=readme-ov-file#open-in-chrome ).

**Comment by Guybrush3pw00d** [comment]
Thanks for your kind and helpful follow-up as always and apologies for my late reply. Eruda is the key as I already expected, thanks for confirming it to me. I need to study how to install and use it: I'll do it ASAP. I've also understood the need for disabling the other browsers (Soul in my case). Thanks for everything and have a nice day! :)
